webapp/lib/model.dart

```diff
@@ -30,7 +30,13 @@ extension ConversationListShardUtil on g.ConversationListShard {
 }
 
 extension ConversationUtil on g.Conversation {
-  String get shortDeidentifiedPhoneNumber => docId.split('uuid-')[1].split('-')[0];
+  String get shortDeidentifiedPhoneNumber {
+    try {
+      return docId.split('uuid-')[1].split('-')[0];
+    } catch (e) {
```

Comment:
Why would this circumstance be triggered?

Reply:
the nook id is now visible in the URL, and so if the url is edited and it no longer meets the "nook-phone-uuid-XXX" format, this will crash. I didn't think this was a very great solution either, so perhaps I should revert this and either catch the error another way, or say, this should never happen - so letting the error through is the right thing to do?